docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
> @csrwng the cacert is used to authenticate the server you're talking to is who it says it is

It looks like it's possible to use a CA cert to verify the server during SSH, but that's not typical. More typical is just to have the server's public key in `~/.ssh/known_hosts`. I guess we can leave ca.crt in place for ssh-privatekey, though we don't have it hooked up to do anything yet